src/docs/releases/README.md

```diff
@@ -0,0 +1 @@
+# Releases
```

Comment:
Will you add links here, or is it auto-generated somehow?

Reply:
I guess I can remove it if we don't use it. 
Not auto-generated.